.github/workflows/chromatic.yml

```diff
@@ -43,7 +46,7 @@ jobs:
       - name: Publish to Chromatic
         uses: chromaui/action@v1
         with:
-          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          projectToken: de095a2cb27a
```

Comment:
is this intentional?

Reply:
Yea: https://www.chromatic.com/docs/custom-ci-provider#run-chromatic-on-external-forks-of-open-source-projects